### ISSUE TITLE
SSH unable to paste when running a function while logging in

### ISSUE THREAD

**Issue by BenRemer** [warpdotdev/Warp#3416]
### Discord username (optional)

_program

### Operating System

MacOS

### Operating System and Version

macOS Ventura Version 13.4 (22F66)

### Local Shell Version

5.9 (zsh)

### Remote Shell Version

5.8.1 (zsh)

### Warp Version

v0.2023.07.18.08.03.stable_00

### I have tried commenting out my system & user rcfiles and warp still will not bootstrap

- [X] Yes

### Screenshots

![Screenshot 2023-07-25 at 11 00 54 AM](https://github.com/warpdotdev/Warp/assets/5099230/83926fc8-9c55-48fe-96ce-8bd4aa4942e1)

I have a function that is called every time I ssh into my host. This function is created in my zshconfig. When I log in I need to give it another password to log into an authentication system we use on the host. On a normal terminal this is taken as normal and I can copy and paste into it but in warp when I try to copy/past it pasts in the `Starting shell...` area and not in the ssh block meaning I cannot paste. 

![Screenshot 2023-07-25 at 11 05 22 AM](https://github.com/warpdotdev/Warp/assets/5099230/36847fe9-7c21-46c3-a044-d03f0eb3d613)

### Include shell xtrace output

Will email as I cannot post publicly. 

### Does this block you from using Warp daily?

No

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore) - linear-label:e7dfaa84-5fdb-4a00-b754-d8912da923fa

None

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
Hey @BenRemer, Thanks for letting us know about this issue. As a workaround, If you can please try and use `/usr/bin/ssh` as a workaround, it should run the ssh session without Warp features, but the paste should work.

To anyone else facing this issue, please add a :+1: to the original post at the top or comment with your details, and subscribe if you'd like to be notified.

**Comment by chadawagner** [comment]
Same here except I'm not ssh'ing anywhere, just running a script from my zshconfig when starting a new terminal window, which sometimes prompts for a TOTP.

**Comment by gibrahim-cr** [comment]
I have a similar issue that is a POC killer, we cant paste into containers we connect to via warp
Example:
`kubectl exec -it <container> -n <namespace> -- bash`

Once I am in the container I cant paste anything from the clipboard.  Its a bizarre behavior.

**Comment by dannyneira** [comment]
Hey Folks, this has been resolved in recent update. Closing as resolved, but please feel free to update Warp and let us know if the issues are resolved.

**Comment by odinxux** [comment]
issue not resolve in ssh 
cant paste string (password) from clip
using command+v throw error (unknown function)as seen in screenshot
warp menu bar has only copy  
<img width="1509" alt="Screenshot 2024-11-28 at 10 36 16" src="https://github.com/user-attachments/assets/d2744c51-8d42-4ab4-be68-c3512b848db3">

**Comment by onbio** [comment]
This issue has been persistant. I had stopped using warp earlier because of this same issue. then I found if we clicked copy working directory or other option the cursor used to come back, but not any more. In the new update if it gets stuck I have no option than closing the terminal. I can't kill the running command in any way or get back where I had to enter.

<img width="1052" height="542" alt="Image" src="https://github.com/user-attachments/assets/8113a39e-b8d2-40d8-a0a4-aad10164abd4" />
